assets/docs/snippets/alb-elb-name.md

```diff
@@ -0,0 +1 @@
+kgateway
```

Comment:
The name that aws autocreates the nlb or alb with is a shortened version of the ns name. in kgw here it uses `kgateway`. in gg it ends up being `gloosyst`. so made a conref for this